src/Microsoft.AspNetCore.Http/HeaderDictionary.cs

```diff
@@ -97,6 +98,34 @@ public HeaderDictionary(int capacity)
             set { this[key] = value; }
         }
 
+        public long? ContentLength
+        {
+            get
```

Comment:
Looks like duplication here and in `DictionaryStringValuesWrapper`. Should we keep this in a helper method like before and just call it from the getter and setter of the `ContentLength`  properties.